### PR TITLE
Add ability to read annotated heartbeats and subject data from SHHS

### PR DESCRIPTION
**Pull request by hofaflo**

### CODE REVIEW

**Comment by cbrnr** [comment]
Do you want to add a changelog entry? Anything else missing or is it ready for merge then?

**Comment by hofaflo** [comment]
Thanks for the reminder, ready to merge :)

File: sleepecg/io/sleep_readers.py [hunk 1]
```diff
@@ -555,15 +579,70 @@ def read_shhs(
...
+                if not offline and rpoints_filename in checksums:
+                    _download_nsrr_file(
+                        download_url + rpoints_filename,
+                        rpoints_filepath,
+                        checksums[rpoints_filename],
+                    )
+                else:
+                    print(f'Skipping {record_id} due to missing heartbeat annotations.')
+                    continue
+            print(record_id)
```
Comment (hofaflo) [inline sleepecg/io/sleep_readers.py:634]:
```suggestion
```

Found a debugging print 🙈

**Comment by cbrnr** [comment]
Thanks @hofaflo!

**Pull request merged**